services/app-api/forms/yearlyFormSelection.test.ts

```diff
@@ -0,0 +1,17 @@
+import { ReportType } from "../types/reports";
+import { qmsReportTemplate as qmsReportTemplate2026 } from "./2026/qms";
+import { getReportTemplate } from "./yearlyFormSelection";
+
+describe("Yearly Form Selection", () => {
+  it("should return the first available template for any year prior to go-live", () => {
+    expect(getReportTemplate(ReportType.QMS, 2025)).toBe(qmsReportTemplate2026);
```

Comment:
Are we currently assuming that for 2025, 2026, 2027 we expect 2026 template (at least for now?)